src/engraving/libmscore/textbase.cpp

```diff
@@ -1668,7 +1668,7 @@ TextBase::TextBase(Score* s, ElementFlags f)
 TextBase::TextBase(const TextBase& st)
     : Element(st)
 {
-    _cursor                      = st._cursor;
+    _cursor                      = new TextCursor(this);
```

Comment:
NB this is actually a memory-leak, but one that already exists when creating new TextBase elements.  As far as I can tell it's completely safe to delete in the destructor, but perhaps we should use unique_ptr<> ?